index.html

```diff
@@ -152,7 +152,7 @@
     </p>
 </section>
 
-<section id="introduction">
+<section class="informative" id="introduction">
```

Comment:
Non-normative/informative sections cannot contain normative subsections.

Therefore, if, as I believe will be the case when PR-Preview does its thing or when this PR is merged, the Terminology section, which is now meant to be normative, is a sub-section of the Introduction section, then the Introduction section *must* be made normative.
```suggestion
<section class="normative" id="introduction">
```